scripts/v120_workflow.py

```diff
@@ -0,0 +1,29 @@
+import compas
+import math
+from compas_view2.flow import Node
+from compas_view2.app import App
+from compas.datastructures import Mesh
+from compas.geometry import Scale, Rotation, Translation
+
+viewer = App(viewmode="shaded", show_flow=True, width=800, height=500)
```

Comment:
would be good if showing the flow can alternatively also be triggered from the toolbar, for example...

Reply:
sure of course! will add that option